pallets/redeem/src/lib.rs

```diff
@@ -819,10 +823,10 @@ impl<T: Config> Pallet<T> {
 			ext::vault_registry::transfer_funds::<T>(
 				CurrencySource::LiquidatedCollateral(vault_id.clone()),
 				slashing_destination,
-				&confiscated_collateral,
+				&vault_to_be_burned_tokens,
 			)?;
 
-			confiscated_collateral
+			vault_to_be_burned_tokens.clone()
```

Comment:
I don't think we need to change this.

Reply:
Yes, I think so, too. The amount given to the function `decrease_liquidated_collateral` should be equal to the one given to `transfer_funds`, otherwise some invariant is violated. This [invariant](https://www.notion.so/satoshipay/Vault-Registry-Pallet-b4a52755953e4db8bdac937be96a7406?pvs=4#a67ed18e9dc549919d05764a71d4e47a) is not mentioned explicitly but must be kept true in my opinion for the vault registry to work correctly.